frontend/components/ActivityDetails/InstallDetails/AppInstallDetails/AppInstallDetails.tsx

```diff
@@ -84,43 +104,71 @@ export const getStatusMessage = ({
   if (displayStatus === "failed_install") {
     return (
       <>
-        The MDM command (request) to install <b>{software_title}</b> on{" "}
-        {formattedHost} failed. Please re-attempt this installation.
+        The MDM command (request) to install <b>{appName}</b>
+        {!isDUP && <> on {formattedHost}</>} failed
+        {!isDUP && displayTimeStamp && <> {displayTimeStamp}</>}. Please
+        re-attempt this installation.
       </>
     );
   }
 
+  const renderSuffix = () => {
+    if (isDUP) {
+      return null;
+    }
+    return (
+      <>
+        {" "}
+        on {formattedHost}
+        {displayStatus === "pending_install" && " when it comes online"}
+        {displayTimeStamp && <> {displayTimeStamp}</>}
+      </>
+    );
+  };
   // Create predicate and subordinate for other statuses
   return (
     <>
-      Fleet {getInstallDetailsStatusPredicate(displayStatus)}{" "}
-      <b>{software_title}</b> on {formattedHost}
-      {displayStatus === "pending" ? " when it comes online" : ""}.
+      Fleet {getInstallDetailsStatusPredicate(displayStatus)} <b>{appName}</b>
+      {renderSuffix()}.
     </>
   );
 };
 
 const baseClass = "app-install-details";
 
-export type IAppInstallDetails = Pick<
-  IActivityDetails,
-  | "host_id"
-  | "command_uuid"
-  | "host_display_name"
-  | "software_title"
-  | "app_store_id"
-  | "status"
-> & {
-  deviceAuthToken?: string;
+export type IVppInstallDetails = {
+  fleetInstallStatus: SoftwareInstallStatus;
+  hostDisplayName: string;
+  appName: string;
+  commandUuid: string;
 };
 
-export const AppInstallDetails = ({
-  status,
-  command_uuid = "",
-  host_display_name = "",
-  software_title = "",
+interface IVPPInstallDetailsModalProps {
+  details: IVppInstallDetails;
+  hostSoftware?: IHostSoftware; // for inventory versions, not present on activity feeds
```

Comment:
Suggestion: we should have a pattern to put comments on the interface above the line as:
` /** for inventory versions, not present on activity feeds */`
so the comments will show up when you hover over the prop on VS code <3